tsc/process/deprecation.md

```diff
@@ -0,0 +1,13 @@
+**Deprecation Strategy for OpenVDB**
+
+OpenVDB is committed to supporting the current and last two years of the VFX Reference Platform.
+
+For example, as of writing it is March 2019 which means VFX Reference Platform 2019, 2018 and 2017
+
+This infers the following support:
+
+* OpenVDB ABI=4, ABI=5 and ABI=6
+* Houdini 16.5, 17.0 and 17.5
```

Comment:
Houdini and Maya aren't actually mentioned in the VFX platform.  Do we need to be explicit about how we choose which app versions to support (i.e., by year of release)?

Reply:
I believe the intent is by whether they support a supported reference platform, rather than their release date.   So if 17.5 supports VFX reference 2018, then it will age out when VDB support for VFX 2018 ages out.

Reply:
Exactly, I'll clarify this point.